bindsnet/datasets/torchvision_wrapper.py

```diff
@@ -0,0 +1,58 @@
+from typing import Optional
+
+import torchvision
+
+from .spike_encoders import Encoder
+
+def vd_wrapper_creator(ds_type):
```

Comment:
What does `vd` stand for?

Reply:
VisionDataset -- this is something that they're working on adding. I was reading their source code when putting this together and not their docs. I'll switch it to something more meaningful for the current implementation in torchvision.